test/testcafe/spec/SelectAuthenticatorForVerification_spec.js

```diff
@@ -298,22 +299,22 @@ test.requestHooks(mockChallengePassword)('should navigate to password challenge
 
   selectFactorPage.selectFactorByIndex(0);
   const challengeFactorPage = new ChallengeFactorPageObject(t);
-  await t.expect(challengeFactorPage.getPageTitle()).eql('Verify with your password');
+  await t.expect(challengeFactorPage.getFormTitle()).eql('Verify with your password');
 });
 
-test.requestHooks(requestLogger, mockChallengePassword)('select password challenge page and hit switch authenticator and re-select password', async t => {
+test.meta('v3', false).requestHooks(requestLogger, mockChallengePassword)('select password challenge page and hit switch authenticator and re-select password', async t => {
```

Comment:
Similar to Enroll, had to disable this because when switching back to authenticator list makes an API request in v3 but does not in V2. Until we update that (which is apart of this ticket https://oktainc.atlassian.net/browse/OKTA-521014) we will need to disable this test for the parity execution.